www_src/pages/project/project.jsx

```diff
@@ -437,6 +437,30 @@ var Project = React.createClass({
     }
   },
 
+  setDestination: function () {
+    // var pageUrl = `/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}`;
+
+    api({
+      method: 'patch',
+      uri: `/users/1/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}/elements/${this.state.routeData.elementID}`,
+      json: {
+        attributes: {
```

Comment:
I wonder if it's possible to update the API to accept JSON "patches" instead of blowing away the entire object every time? @cadecairos any idea?